shub/utils.py

```diff
@@ -48,7 +51,6 @@ def get_key_netrc():
     if key:
         return key
 
-
 def make_deploy_request(url, data, files, auth):
```

Comment:
The other commands besides ```fetch-eggs``` use this method to make requests. That's why I handled the status code also here.